src/Avalonia.Build.Tasks/GenerateAvaloniaResourcesTask.cs

```diff
@@ -38,9 +38,12 @@ public Source(ITaskItem avaloniaResourceItem, string root)
                 _sourcePath = SPath.Combine(root, relativePath);
                 Size = (int)new FileInfo(_sourcePath).Length;
                 var link = avaloniaResourceItem.GetMetadata("Link");
+                var linkBase = avaloniaResourceItem.GetMetadata("LinkBase");
                 var path = !string.IsNullOrEmpty(link)
                     ? link
-                    : relativePath;
+                    : !string.IsNullOrEmpty(linkBase)
```

Comment:
This treats `LinkBase` and `Link` the same, which isn't correct. `Link` is a file path whereas `LinkBase` is the path before the directory globs, i.e. a directory path.